scripts/reindex.py

```diff
@@ -128,7 +122,30 @@ def main(argv: list[str]):
 
     azul = AzulClient(num_workers=args.num_workers)
 
-    source_globs = set(args.sources)
+    if args.catalogs is None:
+        catalog_origin = 'environment variable'
+        if config.current_catalog is None:
+            catalog_origin += ' (unset)'
```

Comment:
> Using catalog(s) specified via environment variable (unset): …

sounds confusing. How can something that's not set specify something? Maybe use a separate, third source to indicate this case?